### PR TITLE
Add Welcome msgs to catalog

### PR DESCRIPTION
**Pull request by stefanalund**
#### :heavy_check_mark: Checklist

- [ ] All tests are passing `yarn test`
- [ ] Screenshots attached (for UI changes)
- [ ] Relevant documentation updated
- [ ] Prettier run on changed files
- [ ] Tests added for new functionality
- [ ] Regression tests added for bug fixes

### CODE REVIEW

File: plugins/catalog/src/components/CatalogPage/utils/timeUtil.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+      greeting: 'Get some rest',
+    };
+  }
+  const timeOfDay = hour => {
+    if (hour < 12) return 'goodMorning';
+    if (hour < 17) return 'goodAfternoon';
+    return 'goodEvening';
+  };
+  // TODO: This does not work.
+  const greetings = require(`utils/locales/${timeOfDay(currentHour)}.locales.json`);
```
Comment (stefanalund) [inline plugins/catalog/src/components/CatalogPage/utils/timeUtil.js:64]: Can someone explain why this does not work? Some webpack magic that I don't understand 🤔

Comment (Rugvip) [reply plugins/catalog/src/components/CatalogPage/utils/timeUtil.js:64]: It's actually lack of magic 😅. We removed src-relative imports so you just have to give a relative path instead. I don't think this dynamic require helps much though tbh, either just import all 3 and select the one we want, or use dynamic imports (´import()´) if we wanna save on the bundle size.

File: plugins/catalog/package.json [hunk 2]
```diff
@@ -30,6 +30,7 @@
     "@material-ui/core": "^4.9.1",
     "@material-ui/icons": "^4.9.1",
     "@material-ui/lab": "4.0.0-alpha.45",
+    "moment": "^2.26.0",
```
Comment (shmidt-i) [inline plugins/catalog/package.json:33]: Are we doing moment and not date-fns for example? Talking about bundle size ;)

Comment (Rugvip) [reply plugins/catalog/package.json:33]: We've kinda just given up and gone with moment. Although tbh big static dependencies aren't that bad given our bundle splitting strategy.

File: plugins/catalog/src/components/CatalogPage/utils/timeUtil.js (hunk 1 above)
Comment (stefanalund) [reply plugins/catalog/src/components/CatalogPage/utils/timeUtil.js]: @Rugvip are you fine this fix?

Comment (Rugvip) [reply plugins/catalog/src/components/CatalogPage/utils/timeUtil.js]: Yup, could make a good first issue about moving it to dynamic imports if you're in the mood for that :grin:

Comment (stefanalund) [reply plugins/catalog/src/components/CatalogPage/utils/timeUtil.js]: For sure! Will create one when this lands.

**Review by Rugvip — APPROVED** [review]

**Pull request merged**